package.json

```diff
@@ -22,6 +22,12 @@
     "test"
   ],
   "author": "Anton <anton@shuvalov.info> (http://shuvalov.info/)",
+  "contributors": [
+    {
+      "name": "Koryukov Maksim",
+      "email": "maxkoryukov@gmail.com"
+    }
+  ],
```

Comment:
Why https://github.com/A/superagent-mocker/graphs/contributors isn't enough for you?